flow/designs/asap7/mock-array/Element/constraints.sdc

```diff
@@ -4,10 +4,11 @@ set cols [expr {[info exists ::env(MOCK_ARRAY_COLS)] ? $::env(MOCK_ARRAY_COLS) :
 
 set clk_name clock
 set clk_port_name clock
-set clk_period 1000
+set clk_period 500
 
 set clk_port [get_ports $clk_port_name]
 create_clock -period $clk_period -waveform [list 0 [expr $clk_period / 2]] -name $clk_name $clk_port
+set_clock_latency -source -100 [get_clocks $clk_name]
```

Comment:
Is it intentional that you are adding more latency at the top level than at the block level?
@louiic does this make sense to you?

Reply:
I was trying to reduce latency at top level to subsume the 100ps network latency within the Element.

Reply:
Basically I am trying to have a globally balanced clock tree across mock-array and Element.